lizard_connector/__init__.py

```diff
@@ -1 +1,4 @@
 # package
+import lizard_connector.jsdatetime
```

Comment:
Dus... je neemt pythonic advies aan van iemand die `from iets import *` durft op te schrijven? :-)

Het is niet pythonic omdat iemand het pythonic noemt.

Ja, het kan. Ja, soms kan het handig zijn. Ja, niemand bij ons op kantoor kijkt standaard in de `__init__.py`.

En volgens mij doen jouw regels ook niet wat je denkt dat ze doen. Ik vermoed dat je op deze manier een `lizard_connector.lizard_connector.query` krijgt als je `import lizard_connector` doet. Danwel een importerror als je `from lizard_connector import query`. Even testen.